dist/obs-server.spec

```diff
@@ -125,6 +126,10 @@ Requires:       perl-Socket-MsgHdr
 Requires:       perl-XML-Parser
 Requires:       perl-XML-Simple
 
+%if %{without image_templates}
+Patch0: hide-image-templates.patch
```

Comment:
It's actually forbidden to have conditional sources - the src.rpm should contain everything